src/components/Hero/Hero.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react'
+import './Hero.scss'
+import { useDispatch } from 'react-redux';
+import { DECREMENT, INCREMENT } from '../../store/actions/types';
+import { decreamentAction, increamentAction } from '../../store/actions/cartActions';
+
+const Hero = (props) => {
```

Comment:
It could be better and also readable to distract the props here and pass them as attributes to the component here, `const Hero = ({ content }) => {`